api/types/sortby_test.go

```diff
@@ -0,0 +1,50 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetSortByFromString(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		in  string
+		out SortBy
+	}{
+		{
+			"hostname:asc",
+			SortBy{
+				Field:  "hostname",
+				IsDesc: false,
+			},
+		},
+		{
+			"",
+			SortBy{},
+		},
+		{
+			"name:desc",
+			SortBy{
+				Field:  "name",
+				IsDesc: true,
+			},
+		},
+		{
+			"hostname",
+			SortBy{
+				Field:  "hostname",
+				IsDesc: false,
+			},
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.in, func(t *testing.T) {
+			t.Parallel()
```

Comment:
I wouldn't bother with the parallel here, personally. It's going to take longer to parallelize this work than to run all 3 subtests on the current goroutine.